From: Markus Niebel <Markus.Niebel@tq-group.com>
Date: Wed, 9 Aug 2017 10:10:38 +0200
Subject: [PATCH] provide a pre-generated penguin.c to get rid of this
 dependency

fbtest compiles pnmtohex for the host. pnmtohex generates
penguin.c from penguin.ppm. pnmtohex needs libnetpbm which
is very painful to compile.

port patch from older release

Signed-off-by: Markus Niebel <Markus.Niebel@tq-group.com>
Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 Makefile         |   2 +-
 images/Makefile  |   3 -
 images/penguin.c | 605 +++++++++++++++++++++++++++++++++++++++++++++++++++++++
 3 files changed, 606 insertions(+), 4 deletions(-)
 create mode 100644 images/penguin.c

diff --git a/Makefile b/Makefile
index 5041f6ca5ca1..512a684f5f32 100644
--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,7 @@ TOPDIR = .
 
 TARGET = $(CROSS_COMPILE)fbtest
 
-SUBDIRS = drawops pnmtohex fonts images visops tests
+SUBDIRS = drawops fonts images visops tests
 
 LIBS += tests/tests.a drawops/drawops.a fonts/fonts.a images/images.a \
 	visops/visops.a
diff --git a/images/Makefile b/images/Makefile
index 2b6b2e27d56b..c098199df710 100644
--- a/images/Makefile
+++ b/images/Makefile
@@ -19,6 +19,3 @@ include $(TOPDIR)/Rules.make
 %.c:	%.pnm
 	../pnmtohex/pnmtohex $< $* > $@
 
-clean::
-	$(RM) $(SRCS)
-
diff --git a/images/penguin.c b/images/penguin.c
new file mode 100644
index 000000000000..e454b99d9505
--- /dev/null
+++ b/images/penguin.c
@@ -0,0 +1,605 @@
+    /*
+     *  Image penguin
+     */
+
+#include "image.h"
+
+static const unsigned char penguin_data[];
+static const unsigned char penguin_clut[];
+
+const struct image penguin = {
+    width:	80,
+    height:	80,
+    type:	IMAGE_CLUT256,
+    data:	penguin_data,
+    clut_len:	187,
+    clut:	penguin_clut
+};
+
+static const unsigned char penguin_data[6400] = {
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x01, 0x01, 0x02, 0x02, 0x02, 0x01, 0x01, 0x01, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07,
+    0x06, 0x06, 0x05, 0x08, 0x03, 0x02, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x09, 0x09, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x01, 0x03, 0x05, 0x0a, 0x0b, 0x0c, 0x0d, 0x0d, 0x0d, 0x0e, 0x0c, 0x0b,
+    0x0a, 0x05, 0x08, 0x02, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x09, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x04, 0x0a, 0x0c, 0x0f,
+    0x0c, 0x10, 0x10, 0x04, 0x05, 0x07, 0x0b, 0x0c, 0x0f, 0x11, 0x12, 0x05,
+    0x03, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x09, 0x09, 0x09, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x02, 0x05, 0x13, 0x14, 0x15, 0x01, 0x16, 0x16, 0x16, 0x16,
+    0x16, 0x16, 0x16, 0x16, 0x01, 0x0b, 0x0f, 0x0c, 0x10, 0x08, 0x01, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x04, 0x13,
+    0x0d, 0x07, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16,
+    0x16, 0x16, 0x01, 0x11, 0x0d, 0x12, 0x04, 0x01, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x09, 0x09, 0x09, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x01, 0x08, 0x0a, 0x14, 0x05, 0x16, 0x16, 0x16,
+    0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x03, 0x12, 0x07, 0x01, 0x16,
+    0x0a, 0x0d, 0x0a, 0x08, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x09, 0x00, 0x09, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x02, 0x06, 0x0c, 0x15, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16,
+    0x16, 0x16, 0x16, 0x05, 0x0f, 0x17, 0x12, 0x02, 0x16, 0x15, 0x11, 0x07,
+    0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x09, 0x09, 0x09, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x03, 0x0a, 0x0f, 0x02,
+    0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x06,
+    0x18, 0x18, 0x15, 0x05, 0x16, 0x01, 0x0d, 0x0b, 0x04, 0x01, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x01, 0x04, 0x19, 0x19, 0x16, 0x16, 0x16, 0x16, 0x16,
+    0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x05, 0x0b, 0x10, 0x08, 0x02,
+    0x16, 0x16, 0x07, 0x14, 0x10, 0x03, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x09, 0x09, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01,
+    0x06, 0x0d, 0x06, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16,
+    0x16, 0x16, 0x16, 0x02, 0x02, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x0d,
+    0x13, 0x08, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x09, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x10, 0x0f, 0x03, 0x16,
+    0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16,
+    0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x0b, 0x0c, 0x05, 0x01, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x09, 0x09, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x03, 0x0a, 0x14, 0x16, 0x16, 0x16, 0x01, 0x02, 0x16,
+    0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x01, 0x03, 0x02, 0x16, 0x16,
+    0x16, 0x16, 0x16, 0x08, 0x14, 0x07, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x09, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x03,
+    0x12, 0x0f, 0x16, 0x16, 0x01, 0x01, 0x04, 0x07, 0x01, 0x16, 0x16, 0x16,
+    0x16, 0x16, 0x08, 0x07, 0x02, 0x13, 0x04, 0x16, 0x16, 0x16, 0x16, 0x02,
+    0x0f, 0x0a, 0x03, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x09, 0x09, 0x09, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x03, 0x12, 0x0f, 0x16, 0x16,
+    0x10, 0x1a, 0x18, 0x04, 0x04, 0x16, 0x16, 0x16, 0x03, 0x0f, 0x1b, 0x1c,
+    0x1d, 0x10, 0x05, 0x01, 0x16, 0x16, 0x16, 0x16, 0x0f, 0x12, 0x03, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x03, 0x12, 0x0f, 0x16, 0x03, 0x1e, 0x1f, 0x20, 0x1a,
+    0x02, 0x16, 0x16, 0x01, 0x21, 0x22, 0x23, 0x24, 0x25, 0x1e, 0x03, 0x01,
+    0x16, 0x16, 0x16, 0x16, 0x0f, 0x13, 0x08, 0x01, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x09, 0x00,
+    0x09, 0x09, 0x09, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x03,
+    0x12, 0x0f, 0x16, 0x0b, 0x24, 0x20, 0x26, 0x27, 0x15, 0x16, 0x16, 0x06,
+    0x23, 0x28, 0x29, 0x2a, 0x2b, 0x2c, 0x0e, 0x16, 0x16, 0x16, 0x16, 0x16,
+    0x11, 0x15, 0x04, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x03, 0x12, 0x14, 0x16, 0x2d,
+    0x2e, 0x05, 0x0f, 0x26, 0x2a, 0x02, 0x03, 0x12, 0x2f, 0x30, 0x01, 0x11,
+    0x31, 0x32, 0x33, 0x16, 0x16, 0x16, 0x16, 0x16, 0x11, 0x15, 0x04, 0x01,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x09, 0x00, 0x09, 0x09, 0x09, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x03, 0x0a, 0x0f, 0x01, 0x1a, 0x2d, 0x01, 0x11, 0x34,
+    0x35, 0x08, 0x10, 0x0b, 0x2b, 0x2d, 0x16, 0x03, 0x12, 0x30, 0x1f, 0x16,
+    0x16, 0x16, 0x16, 0x16, 0x0e, 0x19, 0x04, 0x01, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x09, 0x00,
+    0x09, 0x00, 0x09, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x03,
+    0x0a, 0x18, 0x03, 0x17, 0x35, 0x16, 0x08, 0x1a, 0x36, 0x37, 0x37, 0x38,
+    0x1c, 0x2d, 0x16, 0x16, 0x16, 0x20, 0x20, 0x01, 0x16, 0x16, 0x16, 0x16,
+    0x0e, 0x19, 0x04, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x09, 0x09, 0x09, 0x00, 0x09, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x10, 0x31, 0x03, 0x15,
+    0x23, 0x05, 0x39, 0x3a, 0x3b, 0x3c, 0x3d, 0x3e, 0x3f, 0x40, 0x41, 0x16,
+    0x11, 0x27, 0x1b, 0x16, 0x16, 0x16, 0x16, 0x16, 0x11, 0x0c, 0x05, 0x01,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x02, 0x10, 0x0f, 0x03, 0x02, 0x20, 0x42, 0x43, 0x3d,
+    0x44, 0x45, 0x46, 0x47, 0x48, 0x49, 0x46, 0x3e, 0x4a, 0x4b, 0x0a, 0x16,
+    0x16, 0x16, 0x16, 0x16, 0x13, 0x0e, 0x06, 0x01, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02,
+    0x07, 0x0f, 0x03, 0x16, 0x4c, 0x43, 0x4d, 0x44, 0x3c, 0x46, 0x49, 0x4e,
+    0x4f, 0x50, 0x51, 0x49, 0x49, 0x52, 0x4c, 0x16, 0x16, 0x16, 0x16, 0x16,
+    0x13, 0x14, 0x07, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x07, 0x14, 0x06, 0x53,
+    0x54, 0x55, 0x56, 0x44, 0x45, 0x57, 0x49, 0x58, 0x50, 0x51, 0x51, 0x51,
+    0x52, 0x3f, 0x3e, 0x01, 0x16, 0x16, 0x16, 0x16, 0x05, 0x18, 0x0a, 0x03,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x02, 0x06, 0x0d, 0x13, 0x59, 0x43, 0x5a, 0x5b, 0x3c,
+    0x46, 0x49, 0x4e, 0x5c, 0x51, 0x51, 0x49, 0x5d, 0x5e, 0x5a, 0x5f, 0x16,
+    0x16, 0x16, 0x16, 0x16, 0x01, 0x31, 0x0b, 0x08, 0x01, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02,
+    0x06, 0x0d, 0x12, 0x04, 0x60, 0x61, 0x44, 0x62, 0x57, 0x49, 0x51, 0x51,
+    0x49, 0x63, 0x64, 0x65, 0x5a, 0x65, 0x66, 0x16, 0x01, 0x0b, 0x03, 0x16,
+    0x16, 0x19, 0x0e, 0x06, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x07, 0x0d, 0x13, 0x01,
+    0x67, 0x68, 0x69, 0x52, 0x47, 0x46, 0x3f, 0x69, 0x6a, 0x43, 0x65, 0x6b,
+    0x6c, 0x6d, 0x21, 0x16, 0x16, 0x0d, 0x1a, 0x15, 0x16, 0x04, 0x31, 0x12,
+    0x08, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x02, 0x10, 0x0f, 0x13, 0x01, 0x35, 0x6e, 0x6f, 0x6a,
+    0x5d, 0x3e, 0x70, 0x5e, 0x55, 0x55, 0x70, 0x42, 0x20, 0x1f, 0x29, 0x03,
+    0x16, 0x04, 0x1a, 0x1a, 0x04, 0x16, 0x0e, 0x11, 0x06, 0x02, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x08,
+    0x13, 0x17, 0x05, 0x02, 0x1b, 0x30, 0x6e, 0x6f, 0x70, 0x5e, 0x70, 0x43,
+    0x54, 0x71, 0x72, 0x22, 0x73, 0x2b, 0x25, 0x0c, 0x16, 0x16, 0x13, 0x19,
+    0x01, 0x16, 0x02, 0x31, 0x13, 0x08, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x07, 0x0e, 0x0e, 0x16, 0x01,
+    0x74, 0x1f, 0x30, 0x75, 0x76, 0x6f, 0x6f, 0x77, 0x6e, 0x22, 0x30, 0x23,
+    0x27, 0x28, 0x28, 0x78, 0x01, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x19,
+    0x0e, 0x07, 0x03, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x02, 0x04, 0x0b, 0x18, 0x08, 0x16, 0x12, 0x2c, 0x2b, 0x30, 0x30,
+    0x30, 0x22, 0x22, 0x30, 0x30, 0x20, 0x25, 0x79, 0x28, 0x28, 0x28, 0x28,
+    0x14, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x03, 0x0f, 0x0b, 0x04, 0x01,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x08, 0x12, 0x31,
+    0x12, 0x08, 0x01, 0x78, 0x28, 0x27, 0x7a, 0x30, 0x30, 0x30, 0x30, 0x30,
+    0x7a, 0x2f, 0x7b, 0x28, 0x28, 0x28, 0x28, 0x28, 0x73, 0x03, 0x16, 0x16,
+    0x16, 0x16, 0x16, 0x16, 0x0a, 0x0f, 0x0a, 0x08, 0x01, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x01, 0x03, 0x10, 0x0e, 0x0c, 0x16, 0x01, 0x31, 0x7b,
+    0x28, 0x28, 0x32, 0x1f, 0x30, 0x30, 0x20, 0x2b, 0x27, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x14, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16,
+    0x16, 0x0d, 0x11, 0x07, 0x03, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x03,
+    0x07, 0x0c, 0x0d, 0x01, 0x16, 0x08, 0x24, 0x28, 0x28, 0x28, 0x28, 0x27,
+    0x26, 0x2f, 0x27, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x7c, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x08, 0x31, 0x19,
+    0x06, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x05, 0x15, 0x31, 0x08, 0x16,
+    0x16, 0x7d, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x7b, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x2f, 0x08, 0x16,
+    0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x08, 0x18, 0x0b, 0x05, 0x02, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x01, 0x04, 0x13, 0x31, 0x05, 0x16, 0x16, 0x03, 0x20, 0x7b, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x7b, 0x79, 0x0b, 0x16, 0x16, 0x16, 0x16, 0x16,
+    0x16, 0x16, 0x16, 0x10, 0x0f, 0x13, 0x04, 0x01, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x03, 0x10, 0x14,
+    0x07, 0x16, 0x16, 0x16, 0x0a, 0x20, 0x27, 0x28, 0x28, 0x28, 0x28, 0x7b,
+    0x79, 0x79, 0x7b, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x7b, 0x27, 0x32,
+    0x26, 0x2f, 0x17, 0x01, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16,
+    0x10, 0x14, 0x0a, 0x03, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x02, 0x05, 0x19, 0x0c, 0x16, 0x16, 0x16, 0x01,
+    0x11, 0x2e, 0x7a, 0x2c, 0x27, 0x7b, 0x7b, 0x32, 0x26, 0x2f, 0x32, 0x7b,
+    0x7b, 0x7b, 0x27, 0x2f, 0x25, 0x7a, 0x73, 0x73, 0x1f, 0x73, 0x78, 0x08,
+    0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x19, 0x0c, 0x06,
+    0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x03, 0x0a, 0x14, 0x08, 0x16, 0x16, 0x16, 0x02, 0x18, 0x78, 0x24, 0x79,
+    0x7b, 0x28, 0x28, 0x7b, 0x2c, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x27,
+    0x32, 0x26, 0x23, 0x73, 0x20, 0x20, 0x23, 0x33, 0x01, 0x03, 0x13, 0x03,
+    0x16, 0x16, 0x16, 0x16, 0x16, 0x01, 0x0f, 0x12, 0x08, 0x01, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x04, 0x0b, 0x11, 0x16,
+    0x16, 0x02, 0x16, 0x04, 0x7e, 0x2f, 0x7b, 0x28, 0x28, 0x28, 0x28, 0x7b,
+    0x79, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x27,
+    0x2f, 0x7a, 0x1f, 0x26, 0x18, 0x16, 0x01, 0x10, 0x06, 0x16, 0x16, 0x16,
+    0x16, 0x16, 0x19, 0x0c, 0x05, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x02, 0x06, 0x0e, 0x13, 0x16, 0x05, 0x05, 0x16, 0x2d,
+    0x32, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x27, 0x24, 0x73,
+    0x23, 0x03, 0x16, 0x16, 0x06, 0x04, 0x16, 0x16, 0x16, 0x16, 0x08, 0x0f,
+    0x0a, 0x03, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x03,
+    0x0a, 0x31, 0x04, 0x16, 0x0a, 0x16, 0x08, 0x24, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x7b, 0x2b, 0x24, 0x17, 0x16, 0x03,
+    0x08, 0x10, 0x02, 0x16, 0x16, 0x16, 0x16, 0x0d, 0x15, 0x04, 0x01, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x08, 0x0b, 0x14, 0x16, 0x05,
+    0x04, 0x16, 0x2a, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x7b, 0x32, 0x1f, 0x01, 0x10, 0x15, 0x05, 0x10, 0x16,
+    0x16, 0x16, 0x16, 0x12, 0x0d, 0x06, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x02, 0x06, 0x0e, 0x15, 0x16, 0x0a, 0x16, 0x04, 0x2f, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x7b, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x27, 0x12, 0x10, 0x0a, 0x03, 0x10, 0x03, 0x16, 0x16, 0x16, 0x01,
+    0x0f, 0x12, 0x03, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x03, 0x0a,
+    0x31, 0x08, 0x08, 0x05, 0x16, 0x1a, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x7b, 0x32, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x18, 0x01,
+    0x16, 0x16, 0x02, 0x07, 0x16, 0x16, 0x16, 0x16, 0x0e, 0x15, 0x04, 0x01,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x05, 0x0c, 0x14, 0x16, 0x10, 0x01,
+    0x03, 0x23, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x27, 0x79,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x74, 0x16, 0x16, 0x16, 0x16, 0x12,
+    0x16, 0x16, 0x16, 0x16, 0x0a, 0x0e, 0x06, 0x02, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x01, 0x03, 0x0a, 0x31, 0x05, 0x01, 0x0a, 0x16, 0x0e, 0x7b, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x79, 0x79, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x78, 0x16, 0x16, 0x16, 0x16, 0x12, 0x16, 0x16, 0x16, 0x16,
+    0x02, 0x0f, 0x10, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x05, 0x0c, 0x14,
+    0x16, 0x04, 0x08, 0x16, 0x34, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x2c, 0x79, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x7a, 0x16,
+    0x16, 0x16, 0x16, 0x10, 0x16, 0x16, 0x16, 0x16, 0x01, 0x0f, 0x12, 0x03,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x01, 0x08, 0x12, 0x0f, 0x08, 0x16, 0x07, 0x02, 0x01,
+    0x23, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x2c, 0x79,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x2b, 0x01, 0x16, 0x16, 0x01, 0x06,
+    0x16, 0x16, 0x16, 0x16, 0x16, 0x14, 0x0b, 0x08, 0x01, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02,
+    0x05, 0x0c, 0x19, 0x16, 0x16, 0x10, 0x02, 0x05, 0x32, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x2f, 0x32, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x2f, 0x01, 0x16, 0x16, 0x02, 0x06, 0x16, 0x16, 0x16, 0x16,
+    0x16, 0x0c, 0x15, 0x04, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x10, 0x0d, 0x01, 0x16,
+    0x16, 0x12, 0x03, 0x0a, 0x27, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x2f, 0x79, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x2c, 0x02,
+    0x16, 0x16, 0x04, 0x03, 0x16, 0x16, 0x16, 0x16, 0x16, 0x0c, 0x19, 0x04,
+    0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x01, 0x08, 0x13, 0x0e, 0x16, 0x16, 0x03, 0x11, 0x07, 0x19,
+    0x7b, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x2f, 0x27,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x2c, 0x03, 0x16, 0x16, 0x06, 0x16,
+    0x16, 0x16, 0x16, 0x16, 0x16, 0x0c, 0x19, 0x04, 0x01, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x08,
+    0x0b, 0x19, 0x16, 0x16, 0x16, 0x06, 0x12, 0x11, 0x7b, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x2f, 0x27, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x26, 0x02, 0x16, 0x01, 0x06, 0x16, 0x16, 0x16, 0x16, 0x16,
+    0x16, 0x0c, 0x15, 0x04, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x04, 0x15, 0x19, 0x16, 0x16,
+    0x16, 0x16, 0x06, 0x0d, 0x7b, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x2f, 0x27, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x7a, 0x16,
+    0x04, 0x07, 0x7f, 0x04, 0x05, 0x08, 0x01, 0x16, 0x16, 0x14, 0x0b, 0x08,
+    0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x01, 0x05, 0x19, 0x2d, 0x80, 0x64, 0x61, 0x37, 0x01, 0x19,
+    0x32, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x2f, 0x27,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x33, 0x08, 0x03, 0x16, 0x16, 0x16,
+    0x01, 0x08, 0x0c, 0x10, 0x01, 0x18, 0x13, 0x08, 0x01, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x02, 0x02, 0x08, 0x10,
+    0x0d, 0x81, 0x5a, 0x82, 0x83, 0x83, 0x5f, 0x02, 0x31, 0x32, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x2f, 0x7b, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x84,
+    0x85, 0x85, 0x86, 0x41, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x10, 0x12,
+    0x05, 0x2d, 0x0b, 0x08, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x01, 0x03, 0x04, 0x06, 0x10, 0x13, 0x11, 0x2d, 0x71, 0x3b, 0x82,
+    0x83, 0x83, 0x83, 0x3a, 0x01, 0x0e, 0x26, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x2f, 0x7b, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x87, 0x88, 0x49, 0x46, 0x89,
+    0x16, 0x16, 0x16, 0x16, 0x16, 0x01, 0x06, 0x05, 0x63, 0x8a, 0x0c, 0x05,
+    0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x08, 0x10, 0x15,
+    0x0d, 0x0f, 0x17, 0x2a, 0x40, 0x65, 0x8b, 0x8c, 0x83, 0x83, 0x83, 0x62,
+    0x66, 0x16, 0x12, 0x1f, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x2c, 0x79,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x8d, 0x82, 0x88, 0x8e, 0x8f, 0x16, 0x16, 0x16, 0x16,
+    0x16, 0x16, 0x01, 0x37, 0x57, 0x46, 0x14, 0x07, 0x02, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x03, 0x10, 0x11, 0x90, 0x71, 0x5e, 0x70, 0x70,
+    0x6b, 0x3b, 0x82, 0x83, 0x83, 0x83, 0x83, 0x83, 0x3d, 0x91, 0x16, 0x04,
+    0x33, 0x7b, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x7b, 0x79, 0x8d,
+    0x44, 0x3c, 0x6b, 0x91, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x53, 0x3d,
+    0x62, 0x3c, 0x92, 0x0a, 0x03, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01,
+    0x04, 0x0b, 0x90, 0x6b, 0x3b, 0x56, 0x3b, 0x3b, 0x5b, 0x82, 0x83, 0x83,
+    0x83, 0x83, 0x83, 0x83, 0x88, 0x3e, 0x02, 0x16, 0x01, 0x1a, 0x79, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x2f, 0x1f, 0x93, 0x5b, 0x5b, 0x65, 0x60,
+    0x7f, 0x16, 0x16, 0x16, 0x01, 0x94, 0x5e, 0x5b, 0x44, 0x44, 0x95, 0x15,
+    0x04, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x06, 0x11, 0x96, 0x3b,
+    0x44, 0x82, 0x82, 0x8c, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83,
+    0x83, 0x46, 0x97, 0x16, 0x16, 0x16, 0x0c, 0x2b, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x7a, 0x1f, 0x98, 0x56, 0x56, 0x5a, 0x43, 0x99, 0x9a, 0x66, 0x9a,
+    0x9b, 0x70, 0x3b, 0x44, 0x82, 0x82, 0x9c, 0x0d, 0x07, 0x03, 0x01, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x01, 0x06, 0x0d, 0x71, 0x3b, 0x44, 0x83, 0x83, 0x83,
+    0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x88, 0x63, 0x8f,
+    0x16, 0x16, 0x16, 0x10, 0x24, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x7b, 0x7a, 0x1f, 0x9d,
+    0x3b, 0x5b, 0x9e, 0x65, 0x5e, 0x70, 0x43, 0x70, 0x65, 0x3b, 0x82, 0x83,
+    0x83, 0x8c, 0x3d, 0x95, 0x19, 0x06, 0x03, 0x01, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02,
+    0x06, 0x0d, 0x9f, 0x9e, 0x44, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83,
+    0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x88, 0x68, 0x16, 0x16, 0x16, 0x16,
+    0x0d, 0x7b, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x7b, 0x25, 0x1f, 0xa0, 0x4d, 0x5b, 0x8b, 0x9e,
+    0x5a, 0x6b, 0x6b, 0x5a, 0x3b, 0x44, 0x83, 0x83, 0x83, 0x83, 0x82, 0xa1,
+    0x17, 0x15, 0x06, 0x03, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x06, 0x0e, 0x9f, 0x5a,
+    0x5b, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83,
+    0x83, 0x83, 0x88, 0x52, 0x53, 0x16, 0x16, 0x16, 0x04, 0x32, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x26, 0x22, 0x96, 0x5a, 0x5b, 0x44, 0x5b, 0x56, 0x3b, 0x3b, 0x56,
+    0x5b, 0x82, 0x83, 0x83, 0x83, 0x83, 0x8c, 0x44, 0xa1, 0x2d, 0x0c, 0x07,
+    0x03, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x01, 0x05, 0x11, 0xa2, 0x6b, 0x5b, 0x83, 0x83, 0x83,
+    0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x88,
+    0x69, 0x7f, 0x16, 0x16, 0x12, 0x27, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x2b, 0x0f, 0x6f,
+    0x5a, 0x5b, 0x82, 0x8c, 0x82, 0x44, 0x44, 0x82, 0x82, 0x8c, 0x83, 0x83,
+    0x83, 0x83, 0x83, 0x83, 0x82, 0x3d, 0xa3, 0x0c, 0x06, 0x02, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01,
+    0x05, 0x11, 0xa2, 0x65, 0x5b, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83,
+    0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x46, 0x37, 0x07, 0x2d,
+    0x2b, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x79, 0x14, 0x7f, 0x99, 0x5a, 0x5b, 0x82, 0x83,
+    0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83,
+    0x83, 0x83, 0x82, 0xa2, 0x12, 0x08, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x06, 0x0d, 0xa2, 0x65,
+    0x5b, 0x8c, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83,
+    0x83, 0x83, 0x83, 0x83, 0x88, 0x3f, 0x72, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x24,
+    0x15, 0x16, 0x8f, 0x9b, 0x5a, 0x5b, 0x8c, 0x83, 0x83, 0x83, 0x83, 0x83,
+    0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x8c, 0x82, 0xa0,
+    0x0b, 0x04, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x03, 0x10, 0x0f, 0x96, 0x6b, 0x5b, 0x8c, 0x83, 0x83,
+    0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83,
+    0x83, 0x46, 0x69, 0x25, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x28, 0x28, 0x28, 0x28, 0x28, 0x7b, 0x2e, 0x05, 0x16, 0x16, 0x41, 0x99,
+    0x4d, 0x44, 0x8c, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83,
+    0x83, 0x83, 0x83, 0x83, 0x8c, 0x5b, 0x9e, 0xa3, 0x12, 0x08, 0x01, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x08,
+    0x13, 0xa4, 0x43, 0x9e, 0x82, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83,
+    0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x82, 0x52, 0x61, 0xa5,
+    0x26, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28, 0x28,
+    0x1f, 0x0c, 0x16, 0x16, 0x16, 0x16, 0xa6, 0x6f, 0x4d, 0x44, 0x8c, 0x83,
+    0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x82,
+    0x8b, 0x6b, 0x90, 0x0c, 0x06, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x04, 0x15, 0x76, 0x55, 0x8b,
+    0x82, 0x8c, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83,
+    0x83, 0x83, 0x83, 0x83, 0x8c, 0x5b, 0x61, 0x99, 0x53, 0x1b, 0x24, 0x7b,
+    0x28, 0x28, 0x28, 0x7b, 0x79, 0x23, 0x74, 0x0c, 0x01, 0x16, 0x16, 0x16,
+    0x16, 0x16, 0x53, 0x99, 0x5a, 0x5b, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83,
+    0x83, 0x83, 0x83, 0x83, 0x83, 0x44, 0x56, 0x5a, 0x71, 0x95, 0x11, 0x10,
+    0x08, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x01, 0x04, 0x19, 0x77, 0x55, 0x9e, 0x5b, 0x44, 0x82, 0x82,
+    0x8c, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83,
+    0x83, 0x5b, 0x5a, 0xa7, 0xa8, 0x16, 0x01, 0x06, 0x0b, 0x19, 0x13, 0x10,
+    0x03, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x01, 0xa8, 0x9b,
+    0x6b, 0x5b, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x82, 0x44, 0x44,
+    0x56, 0x65, 0x9f, 0x95, 0x14, 0x0b, 0x07, 0x08, 0x01, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x08,
+    0x13, 0xa9, 0x43, 0x5e, 0x5a, 0x4d, 0x9e, 0x3b, 0x56, 0x5b, 0x44, 0x44,
+    0x82, 0x8c, 0x83, 0x83, 0x83, 0x83, 0x83, 0x83, 0x8c, 0x56, 0x65, 0x99,
+    0x59, 0x02, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16,
+    0x16, 0x16, 0x16, 0x16, 0x16, 0x01, 0xaa, 0x9b, 0x55, 0x56, 0x82, 0x83,
+    0x83, 0x83, 0x8c, 0x82, 0x44, 0x56, 0x9e, 0x6b, 0x96, 0x95, 0x0f, 0x15,
+    0x10, 0x04, 0x02, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x03, 0x07, 0x11, 0xab, 0xa9,
+    0x9b, 0x54, 0x43, 0x70, 0x5e, 0x55, 0x6b, 0x4d, 0x9e, 0x56, 0x44, 0x82,
+    0x8c, 0x8c, 0x8c, 0x8c, 0x44, 0x5a, 0x64, 0xac, 0x59, 0x7f, 0x16, 0x16,
+    0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16, 0x16,
+    0x16, 0x01, 0xa8, 0xac, 0x43, 0x4d, 0x5b, 0x44, 0x8c, 0x82, 0x44, 0x5b,
+    0x9e, 0x55, 0x43, 0x76, 0x18, 0x19, 0x0a, 0x04, 0x03, 0x01, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x01, 0x08, 0x07, 0x15, 0x0d, 0x21, 0x95, 0xa5, 0x6f,
+    0x99, 0x9b, 0xa7, 0x54, 0x64, 0x70, 0x65, 0x4d, 0x3b, 0x5b, 0x5b, 0x8b,
+    0x4d, 0x70, 0x99, 0xad, 0xaa, 0x02, 0x16, 0x16, 0x08, 0x10, 0x10, 0x10,
+    0x10, 0x10, 0x10, 0x10, 0x10, 0x10, 0x05, 0x16, 0x16, 0x01, 0x94, 0x60,
+    0xa7, 0x5e, 0x4d, 0x56, 0x8b, 0x56, 0x4d, 0x65, 0x43, 0x99, 0x95, 0x14,
+    0x13, 0x06, 0x03, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x01, 0x03, 0x04, 0x07, 0x0a, 0x15, 0x0e, 0x0f, 0x21, 0xae, 0xaf, 0x4c,
+    0x60, 0xac, 0x99, 0x54, 0x64, 0x70, 0x55, 0x5e, 0x54, 0x6f, 0xad, 0x59,
+    0xa6, 0x0b, 0x7d, 0x21, 0x0f, 0x14, 0x0d, 0x0d, 0x0d, 0x0d, 0x0d, 0x0d,
+    0x0d, 0x14, 0x0f, 0x18, 0x2d, 0x17, 0xb0, 0xb1, 0x6f, 0x54, 0x43, 0x5e,
+    0x55, 0x5e, 0x43, 0xa7, 0x68, 0xa4, 0x11, 0x0a, 0x04, 0x02, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x02,
+    0x03, 0x04, 0x06, 0x10, 0x13, 0x19, 0x0e, 0x31, 0x21, 0x92, 0x4c, 0xb1,
+    0x60, 0xac, 0xac, 0xac, 0xb2, 0xb1, 0x97, 0xb3, 0x21, 0x14, 0x15, 0x12,
+    0x10, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x10, 0x0a,
+    0x0b, 0x14, 0xb4, 0xaa, 0xb5, 0x6f, 0x9b, 0xa7, 0xa7, 0x9b, 0xac, 0x4c,
+    0x21, 0x19, 0x07, 0x08, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x01, 0x02,
+    0x08, 0x04, 0x06, 0x0a, 0x13, 0x0c, 0x0f, 0x21, 0xb6, 0x97, 0x59, 0x59,
+    0x59, 0xaa, 0xb7, 0x31, 0x19, 0x10, 0x04, 0x03, 0x02, 0x02, 0x02, 0x02,
+    0x02, 0x02, 0x01, 0x02, 0x02, 0x02, 0x02, 0x03, 0x04, 0x0a, 0x11, 0xb8,
+    0xa8, 0x59, 0xb1, 0x60, 0xb5, 0x9a, 0xb9, 0x0f, 0x15, 0x06, 0x03, 0x01,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x02, 0x03,
+    0x08, 0x05, 0x10, 0x0b, 0x11, 0x0f, 0xb4, 0xb8, 0xb8, 0x0f, 0x0e, 0x13,
+    0x06, 0x03, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x01, 0x08, 0x07, 0x15, 0x14, 0xb8, 0xb8, 0xb8,
+    0xba, 0xb4, 0x0e, 0x13, 0x05, 0x03, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x01, 0x03, 0x08,
+    0x06, 0x10, 0x12, 0x0b, 0x13, 0x0a, 0x06, 0x08, 0x02, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x01, 0x03, 0x05, 0x10, 0x13, 0x15, 0x15, 0x0b, 0x0a, 0x06, 0x08,
+    0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x01, 0x02, 0x03, 0x08,
+    0x08, 0x03, 0x02, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01,
+    0x03, 0x08, 0x04, 0x04, 0x08, 0x03, 0x02, 0x01, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00,
+};
+
+static const unsigned char penguin_clut[561] = {
+    0x00, 0x00, 0x00, 0x06, 0x06, 0x06, 0x0a, 0x0a, 0x0a, 0x0e, 0x0e, 0x0e,
+    0x16, 0x16, 0x16, 0x1a, 0x1a, 0x1a, 0x1e, 0x1e, 0x1e, 0x22, 0x22, 0x22,
+    0x12, 0x12, 0x12, 0x00, 0x00, 0x01, 0x2a, 0x2a, 0x2a, 0x36, 0x36, 0x36,
+    0x42, 0x42, 0x42, 0x4e, 0x4e, 0x4e, 0x4a, 0x4a, 0x4a, 0x56, 0x56, 0x56,
+    0x26, 0x26, 0x26, 0x46, 0x46, 0x46, 0x2e, 0x2e, 0x2e, 0x32, 0x32, 0x32,
+    0x52, 0x52, 0x52, 0x3a, 0x3a, 0x3a, 0x02, 0x02, 0x06, 0x65, 0x65, 0x65,
+    0x5e, 0x5e, 0x5e, 0x3e, 0x3e, 0x3e, 0x74, 0x74, 0x74, 0x8a, 0x8a, 0x8a,
+    0xa2, 0xa2, 0xa2, 0x9a, 0x9a, 0x9a, 0x86, 0x86, 0x86, 0xc6, 0xc6, 0xc6,
+    0xc3, 0xc3, 0xc3, 0x65, 0x62, 0x59, 0xbb, 0xbb, 0xbb, 0xd2, 0xd2, 0xd2,
+    0xda, 0xda, 0xda, 0xd6, 0xd6, 0xd6, 0xe2, 0xe2, 0xe2, 0xf6, 0xf6, 0xf6,
+    0xfd, 0xfd, 0xfd, 0xae, 0xae, 0xae, 0x7b, 0x7b, 0x7b, 0xdd, 0xdd, 0xdd,
+    0xea, 0xea, 0xea, 0x6a, 0x6a, 0x6a, 0xaa, 0xaa, 0xaa, 0xe7, 0xe7, 0xe7,
+    0xbe, 0xbe, 0xbe, 0x5a, 0x5a, 0x5a, 0xee, 0xee, 0xee, 0x9e, 0x9e, 0x9e,
+    0x95, 0x95, 0x95, 0x80, 0x80, 0x80, 0x76, 0x62, 0x2e, 0x79, 0x5c, 0x08,
+    0x62, 0x4e, 0x0a, 0x36, 0x26, 0x06, 0x9a, 0x72, 0x0a, 0xe2, 0xaa, 0x0b,
+    0xec, 0xba, 0x0b, 0xe1, 0xaf, 0x0f, 0xb8, 0x90, 0x0c, 0xd7, 0xae, 0x0f,
+    0xaf, 0x92, 0x3d, 0x25, 0x1a, 0x09, 0xbc, 0xa4, 0x73, 0xc0, 0x85, 0x09,
+    0xef, 0xb6, 0x0d, 0xea, 0xbe, 0x0a, 0xe8, 0xc3, 0x10, 0xe8, 0xc8, 0x1e,
+    0xf5, 0xcf, 0x2d, 0xf1, 0xd0, 0x13, 0xda, 0xc2, 0x86, 0xd3, 0xce, 0xba,
+    0x79, 0x57, 0x19, 0xdb, 0xa2, 0x0a, 0xf4, 0xd6, 0x36, 0xf6, 0xda, 0x3c,
+    0xf6, 0xda, 0x26, 0xf6, 0xd7, 0x14, 0xe2, 0xb8, 0x0d, 0x3d, 0x2a, 0x06,
+    0xb4, 0x7b, 0x07, 0xce, 0x91, 0x0a, 0xe6, 0xae, 0x0b, 0xee, 0xca, 0x0f,
+    0xf6, 0xda, 0x4a, 0x68, 0x45, 0x06, 0xd8, 0x9e, 0x0a, 0xec, 0xb2, 0x0c,
+    0xf5, 0xd7, 0x2b, 0xc6, 0x9b, 0x0a, 0xc8, 0x90, 0x0b, 0x9c, 0x76, 0x0a,
+    0x89, 0x5c, 0x06, 0xd2, 0xa2, 0x0a, 0xee, 0xbe, 0x0a, 0xcb, 0xa6, 0x11,
+    0xb9, 0x85, 0x0b, 0xd2, 0x96, 0x0a, 0x66, 0x4e, 0x0a, 0x5e, 0x46, 0x1e,
+    0x8b, 0x66, 0x0f, 0xbe, 0x92, 0x0d, 0xa8, 0x7a, 0x0a, 0xd5, 0x9a, 0x0b,
+    0xca, 0x96, 0x22, 0xb6, 0x9d, 0x6a, 0xae, 0x9a, 0x72, 0x9c, 0x6b, 0x0b,
+    0xc5, 0x8a, 0x0b, 0xbe, 0x8e, 0x22, 0xbe, 0xb2, 0x90, 0xca, 0xca, 0xca,
+    0x90, 0x90, 0x90, 0xb2, 0xa6, 0x92, 0x9a, 0x79, 0x3c, 0xa8, 0x7c, 0x2c,
+    0xb6, 0xb6, 0xb6, 0xf2, 0xf2, 0xf2, 0xce, 0xce, 0xce, 0xfa, 0xfa, 0xfa,
+    0xb2, 0xb2, 0xb2, 0x6e, 0x6e, 0x6e, 0xa6, 0xa6, 0xa6, 0x12, 0x0e, 0x06,
+    0x4a, 0x36, 0x0e, 0x8e, 0x86, 0x6a, 0xf2, 0xba, 0x0e, 0xf6, 0xbe, 0x0e,
+    0xf6, 0xe6, 0xbe, 0xee, 0xcc, 0x5b, 0xb5, 0x8e, 0x2c, 0xe4, 0xb8, 0x3e,
+    0xf1, 0xc4, 0x0e, 0x26, 0x1e, 0x0a, 0x9a, 0x8e, 0x5a, 0xea, 0xae, 0x0d,
+    0xf6, 0xba, 0x0e, 0xe0, 0xb2, 0x3e, 0xd2, 0xa6, 0x0a, 0x16, 0x12, 0x06,
+    0x9a, 0x7a, 0x2e, 0x2e, 0x20, 0x06, 0x70, 0x64, 0x4e, 0xd6, 0xaa, 0x36,
+    0x46, 0x2f, 0x06, 0x7c, 0x70, 0x58, 0xb4, 0x85, 0x24, 0x62, 0x46, 0x06,
+    0xd6, 0xa6, 0x3a, 0xa3, 0x6e, 0x08, 0x74, 0x51, 0x08, 0xa7, 0x72, 0x07,
+    0xa2, 0x92, 0x5e, 0xca, 0xa2, 0x45, 0xe0, 0xa6, 0x0a, 0xae, 0x87, 0x32,
+    0xbe, 0x96, 0x2e, 0xce, 0xa2, 0x2a, 0xa3, 0x85, 0x43, 0x8e, 0x7a, 0x48,
+    0x6d, 0x6a, 0x5f, 0x8e, 0x6e, 0x2e, 0x32, 0x22, 0x06, 0xaf, 0x76, 0x06,
+    0x50, 0x36, 0x07, 0x9e, 0x76, 0x24, 0x5b, 0x3c, 0x06, 0x8a, 0x6e, 0x32,
+    0x98, 0x63, 0x06, 0x82, 0x53, 0x06, 0x7a, 0x66, 0x46, 0x82, 0x62, 0x2e,
+    0x56, 0x42, 0x22, 0x7c, 0x50, 0x06, 0x8a, 0x56, 0x06, 0x56, 0x42, 0x1e,
+    0x5e, 0x56, 0x4c, 0x86, 0x56, 0x06, 0x6a, 0x56, 0x3a, 0x52, 0x3e, 0x22,
+    0x59, 0x51, 0x42, 0x64, 0x52, 0x34, 0x5e, 0x56, 0x42,
+};
+
